mariadb_config/CMakeLists.txt

```diff
@@ -52,6 +52,10 @@ CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/libmariadb.pc.in
 
 ADD_EXECUTABLE(mariadb_config ${CMAKE_CURRENT_BINARY_DIR}/mariadb_config.c)
 
+IF(CMAKE_SYSTEM_NAME MATCHES AIX)
+  TARGET_LINK_LIBRARIES(mariadb_config compat-getopt)
```

Comment:
@EGuesnet what provides the compat-getopt library?

Reply:
The getopt_long() function is not available on AIX by default. Opensource teams on AIX have created compatibility libraries to provide some function.

Reply:
where is it available?

Reply:
The [lib](http://www.bullfreeware.com/pkg?id=5803), and [devel](http://www.bullfreeware.com/pkg?id=5804) package. They are not available yet on _AIX Toolbox for Linux Applications._